lib/prefab/criteria_evaluator.rb

```diff
@@ -137,13 +137,42 @@ def PROP_AFTER(criterion, properties)
       evaluate_date_comparison(criterion, properties, :>).matched
     end
 
+    def PROP_SEMVER_LESS_THAN(criterion, properties)
+      evaluate_semver_comparison(criterion, properties, COMPARE_TO_OPERATORS[:less_than]).matched
+    end
+
+    def PROP_SEMVER_EQUAL(criterion, properties)
+      evaluate_semver_comparison(criterion, properties, COMPARE_TO_OPERATORS[:equal_to]).matched
+    end
+
+    def PROP_SEMVER_GREATER_THAN(criterion, properties)
+      evaluate_semver_comparison(criterion, properties, COMPARE_TO_OPERATORS[:greater_than]).matched
+    end
 
     def value_from_properties(criterion, properties)
       criterion.property_name == NAMESPACE_KEY ? @namespace : properties.get(criterion.property_name)
     end
 
+    COMPARE_TO_OPERATORS = {
+      less_than_or_equal: -> cmp {  cmp <= 0 },
+      less_than: -> cmp {  cmp < 0 },
+      equal_to: -> cmp {  cmp == 0 },
+      greater_than: -> cmp {  cmp > 0 },
+      greater_than_or_equal: -> cmp {  cmp >= 0 },
+    }
```

Comment:
Small ruby best practice thing: you generally want to throw a `.freeze` at the end of this. 

By far one of the goofiest things in the language is that you can reassign constants (you get a warning, but nothing stops you). If you try to modify a frozen object, you'll get a proper error. I have no reason to believe this'll get modified in normal usage, but it would be noteworthy if it did.